lib/views/home_content/map/post_map.dart

```diff
@@ -15,13 +17,28 @@ class PostMap extends StatelessWidget {
   });
 
   @override
-  Widget build(BuildContext context) {
+  Widget build(BuildContext context, WidgetRef ref) {
+    final currentPosition = ref.watch(liveLocationServiceProvider);
+    final positionNotifier = ref.watch(mapProvider.notifier);
+
+    currentPosition.when(
+      data: (data) {
+        debugPrint("Live location: ${data!.latitude}, ${data.longitude}");
+        positionNotifier.animateCam(LatLng(data.latitude, data.longitude));
```

Comment:
As a user I found it quite frustrating that the camera got reset when I moved. Is this really necessary? I think best is to remove this and just have the recenter button to reset.

If we do want to keep this, I think we should disable gestures in the map, and keep it always centered on the current position. I think this would be worse however.

Reply:
Fixed with 8a540dc